.travis.yml

```diff
@@ -80,7 +80,7 @@ matrix:
 
 env:
   global:
-    - BUILD_TYPE=Debug ASAN=On CTEST_OUTPUT_ON_FAILURE=1
+    - BUILD_TYPE=Debug ASAN=On CTEST_OUTPUT_ON_FAILURE=1 LSAN_OPTIONS=verbosity=1:log_threads=1
```

Comment:
nit: if you would like, let's separate this change to another diff. otherwise it is ok.

Reply:
(y) this is split out into #836